packages/react/src/lib/accessibility/Behaviors/Attachment/attachmentBehavior.ts

```diff
@@ -10,6 +10,7 @@ const attachmentBehavior: Accessibility = (props: any) => ({
   attributes: {
     root: {
       tabIndex: 0,
+      'data-is-focusable': true,
```

Comment:
As I remember we're using `IS_FOCUSABLE_ATTRIBUTE`:

```suggestion
      [IS_FOCUSABLE_ATTRIBUTE]: true,
```

Reply:
truee, thanks! 💃